packages/tools/devtools/devtools-core/src/IContainerDevtools.ts

```diff
@@ -20,44 +19,7 @@ import { AudienceChangeLogEntry, ConnectionStateChangeLogEntry } from "./Logs";
  *
  * @internal
  */
-export interface IContainerDevtools extends IDisposable {
-	/**
-	 * The ID of {@link IContainerDevtools.container}.
-	 */
-	readonly containerId: string;
-
-	/**
-	 * The Container session with which the debugger is associated.
-	 */
-	readonly container: IContainer;
```

Comment:
These other four were just not used?

Reply:
Not since we switched the view library to message-passing. These are required properties as implementation details of the class, but no consumers of the interface need these.